### DISCUSSION TITLE
Doc Question

### DISCUSSION THREAD

**Discussion by icbd** [python-poetry/poetry#9818] [Q&A] [answered]
### Issue Kind

Unclear documentation

### Existing Link

https://python-poetry.org/docs/pyproject/#extras

### Description

Doc link: https://python-poetry.org/docs/pyproject/#extras

> Any extras you don’t specify will be removed. Note this behavior is different from [optional dependency groups](https://python-poetry.org/docs/managing-dependencies/#optional-groups) not selected for install, e.g. those not specified via `install --with`.

What exactly does "removed" refer to here? Can someone help explain "different" in more detail here? 

Thanks

**Comment by clintonroy** [comment, marked as answer]
if your root package has two extras, say FOO and BAR, if you install with extras FOO, then install with extras BAR, everything associated with extra FOO will be removed.
